### PR TITLE
Scene 6

### PR DESCRIPTION
**Pull request by vladiAN**
Added scene with triangle

### CODE REVIEW

**Comment by miroshniktyt** [comment]
one combed please:)

**Comment by vladiAN** [comment]
done

File: concentration/Scene/TriangleScene.swift [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        self.physicsBody = borderBody
+        
+        
+        addTriangle()
+    }
+    
+    func setTriangle() -> SKSpriteNode {
+        
+        let triangleTexture = SKTexture(imageNamed: "triangle")
+        let spriteTriangle = SKSpriteNode(texture: triangleTexture, size: .init(width: self.frame.width - (sizeBall * 2), height: 100))
```
Comment (miroshniktyt) [inline concentration/Scene/TriangleScene.swift]: Make a bit smaller so there is a bit more space then just ball size (maybe putting 4 points spacing on each side)

File: concentration/Scene/TriangleScene.swift [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+//  concentration
+//
+//  Created by Vlad Andrushok on 30.01.2023.
+//
+
+import SpriteKit
+
+class TriangleScene: SKScene {
+    
+    let sizeBall: CGFloat = 30
```
Comment (miroshniktyt) [inline concentration/Scene/TriangleScene.swift]: Make it bigger, at least 44point, also make it depend in balls number (if there are 100 balls let size be 44, but is there are only 8 balls, make them bigger

File: concentration/Scene/TriangleScene.swift [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        
+        for _ in 0...50 {
+            let copy = spriteСircul.copy() as! SKSpriteNode
+            self.addChild(copy)
+        }
+
+        
+        self.backgroundColor = .cyan
+        
+        let borderBody =  SKPhysicsBody (edgeLoopFrom: self.frame)
```
Comment (miroshniktyt) [inline concentration/Scene/TriangleScene.swift:42]: but margins on the button so balls don't go under safe areas, also don't forget that it is different on iPhone se

**Review by miroshniktyt — CHANGES_REQUESTED** [review]

File: concentration/Scene/TriangleScene.swift (hunk 2 above)
Comment (vladiAN) [reply concentration/Scene/TriangleScene.swift]: done

File: concentration/Scene/TriangleScene.swift (hunk 3 above)
Comment (vladiAN) [reply concentration/Scene/TriangleScene.swift:42]: done

**Review by miroshniktyt — APPROVED** [review]

File: concentration/Scene/TriangleScene.swift (hunk 3 above)
Comment (vladiAN) [reply concentration/Scene/TriangleScene.swift:42]: 🫣sorry, but I just now saw the third comment, and made edits only on the first two.

**Pull request merged**